workflows/subwf-atac-single-organism.wdl

```diff
@@ -130,6 +134,10 @@ workflow wf_atac {
         File? share_atac_archr_arrow = archr.archr_arrow
         File? share_atac_archr_obj = archr.archr_obj
         File? share_atac_archr_plots_zip = archr.plots_zip
-        
+        Int atac_total_reads = log_atac.atac_total_reads
+        Int atac_aligned_uniquely = log_atac.atac_aligned_uniquely
+        Int atac_unaligned = log_atac.atac_unaligned
+        Int atac_feature_reads = log_atac.atac_feature_reads
+        Int atac_duplicate_reads = log_atac.atac_duplicate_reads        
```

Comment:
The naming is not consistent with the other outputs
atac_total_reads
instead of:
share_atac_total_reads